### DISCUSSION TITLE
Exit Buttons Function Design

### DISCUSSION THREAD

**Discussion by ZiyangWang001007081** [ZiyangWang001007081/Tafe230243#6] [General]
Hi Team,

I have noticed there are multiple Exit buttons in our program.

![image](https://github.com/ZiyangWang001007081/Tafe230243/assets/151332477/33c6da0b-ec67-4b26-9936-661fcd3e1292)
![image](https://github.com/ZiyangWang001007081/Tafe230243/assets/151332477/159d0749-5219-4c19-b399-e32fd5553767)

The Exit button on the MainMenu would led the user exit the **whole application**. 
However, what function suppose the other Exit buttons on CurrencyConverter and MortageCalculator have? 
Should it to _exit the whole application_, or _exit to the MainMenu Page_ so the user can further select other functions?

Please share your thoughts. Thank you.

Ziyang

**Comment by Abhishek-Sathe-Tafe** [comment]
I think the 'Exit' button on the mortgage or currency calculator should return the user to the main menu. Thanks!

**Reply by ZiyangWang001007081** [reply]
Agreed! I have set my exit button navigate to MainMenu now.
